### DISCUSSION TITLE
Nuxt support

### DISCUSSION THREAD

**Discussion by wobsoriano** [clerk/javascript#1430] [Feature Requests]
I published an [H3 middleware](https://github.com/wobsoriano/h3-clerk) for Clerk, which will let you integrate Clerk in a Nuxt application. Here's a [demo](https://github.com/wobsoriano/nuxt-clerk-playground) with Nuxt 3.

The H3 middleware basically is a port of the Fastify `withClerkMiddleware` middleware.

Is the Clerk team interested in a PR? I can add it to https://github.com/clerkinc/javascript/tree/main/packages/h3 with test and everything 👍🏼 - this should attract more Vue/Nuxt users 😄

**Comment by jescalan** [comment]
👋 hey rob! so sorry for the late response here - this is awesome and we have your vue integration to our docs! 🙌
